tests/nn/moe/test_moe_layer.py

```diff
@@ -135,6 +135,31 @@ def test_forward_routing(device):
         assert torch.allclose(input[:, i] * (expert + 1), output[:, i])
 
 
+@pytest.mark.mpi
+@pytest.mark.parametrize("device", ["cpu"])
+def test_forward_routing_multi(device):
+    model_dim = 8
+    num_local_experts = 4
+    num_experts = dist.get_world_size(dist.group.WORLD) * num_local_experts
```

Comment:
Is it possible to set `world_size=4` in `setup_module(module)`?

Reply:
I will work on making that possible. Currently I run this:

 % mpirun -n 4 pytest --only-mpi tests/nn/moe/test_moe_layer.py

Reply:
You will need to install the following from requirements-test.txt:

mpi4py == 3.0.3
pytest-mpi == 0.4